src/components/LegendModal.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div id="legendModal">
+    <div id="legendModalExit">
```

Comment:
Do we want the click event on the icon or on the 'div' containing the icon.

Reply:
I would say the link is perfect, as the div currently stretches the whole length of the modal, so an adherent click on the left side of the modal would close the info if we went that route.

Reply:
Which would probably lead to confusion

Reply:
Totally trust you, dude.